packages/app-mobile/components/screens/Notes.tsx

```diff
@@ -238,7 +238,12 @@ class NotesScreenComponent extends BaseScreenComponent<any> {
 		const thisComp = this;
 
 		const makeActionButtonComp = () => {
-			if (this.props.notesParentType === 'Folder' && itemIsInTrash(parent)) return null;
+			const trashFolderId = getTrashFolderId();
+
+			// calculate number of folders except deleted and trash folder
+			const atleastOneNotebookExists = this.props.folders.filter((folder: FolderEntity) => folder.id !== trashFolderId && folder.deleted_time === 0).length > 0;
```

Comment:
Since you use the same code in two places please make it a function that takes folders as input and returns a boolean. You can put it in models/Folder.ts

Also name it something like `atLeastOneRealFolderExists` since the trash folder is a virtual one

Reply:
I made the changes you suggested and have added unit test for the same.